src/intTest/resources/success_uat_data/NHS002/full_acceptance_without_nhsack.1.fhir.json

```diff
@@ -0,0 +1,131 @@
+{
+  "resourceType": "Bundle",
+  "id": "cb0cc95f-79c2-468c-acda-463cb6b5467a",
+  "meta": {
+    "lastUpdated": "2021-03-19T14:43:17.005+00:00",
+    "profile": [
+      "https://fhir.nhs.uk/STU3/StructureDefinition/ITK-Message-Bundle-1"
+    ]
+  },
+  "identifier": {
+    "system": "https://tools.ietf.org/html/rfc4122",
+    "value": "26ed0510-a157-41b3-b5c7-71088575fb86"
+  },
+  "type": "message",
+  "entry": [
+    {
```

Comment:
Is there meant to be an `Organization` FHIR resource here? 

The corresponding EDIFACT has two `NAD+SLA` segments (a `DEPT` and `ORG`).

Reply:
There is one, it's just below. Line 53